openstf/model/confidence_interval_applicator.py

```diff
@@ -137,3 +147,24 @@ def _add_quantiles_to_forecast(
             )
 
         return forecast
+
+    def _add_quantiles_to_forecast_quantile_regression(
+        self, forecast: pd.DataFrame, quantiles: list
+    ) -> pd.DataFrame:
+        """Add quantiles to forecast.
+        Use trained quantile regression model to calculate the quantiles.
+        Args:
+            forecast (pd.DataFrame): Forecast
+            quantiles (list): List with desired quantiles
+        Returns:
+            (pd.DataFrame): Forecast DataFrame with quantile (e.g. 'quantile_PXX')
+                columns added.
+        """
+
+        for quantile in quantiles:
+            quantile_key = f"quantile_P{quantile * 100:02.0f}"
+            forecast[quantile_key] = self.model.predict(
+                self.forecast_input_data.sort_index(axis=1), quantile=quantile
```

Comment:
Why do we need to call `sort_index(axis=1)` here? Why is the input data not already in the right format? If it makes sense that the input data is not in the right format I would suggest to do this formatting in a separate step.